src/pages/view/ViewDashboard.js

```diff
@@ -15,191 +17,138 @@ import {
-            loadDashboard()
-        } else if (!requestedIsAvailable && switchingDashboard) {
-            setLoaded(false)
-            props.setSelectedAsOffline(props.requestedId, props.username)
-        }
-    }, [props.requestedId, props.currentId, loaded, online])
-
-    const onExpandedChanged = (expanded) => setControlbarExpanded(expanded)
-
-    const getContent = () => {
-        if (
-            !online &&
-            !isCached &&
-            (props.requestedId !== props.currentId || !loaded)
-        ) {
-            return (
-                <Notice
-                    title={i18n.t('Offline')}
-                    message={
-                        <>
-                            <p>
-                                {i18n.t(
-                                    'This dashboard cannot be loaded while offline.'
-                                )}
-                            </p>
-                            <div>
-                                <Link
-                                    to={ROUTE_START_PATH}
-                                    className={classes.link}
-                                >
-                                    {i18n.t('Go to start page')}
-                                </Link>
-                            </div>
-                        </>
-                    }
-                />
-            )
-        }
-
-        if (loadFailed) {
-            return (
-                <Notice
-                    title={i18n.t('Load dashboard failed')}
-                    message={i18n.t(
-                        'This dashboard could not be loaded. Please try again later.'
-                    )}
-                />
-            )
-        }
-
-        return props.requestedId !== props.currentId ? (
-            <LoadingMask />
-        ) : (
-            <>
-                <TitleBar />
-                <FilterBar />
-                <ItemGrid />
-            </>
-        )
-    }
+    }, [
+        engine,
+        online,
+        passiveViewRegistered,
+        registerPassiveView,
+        requestedId,
+    ])
+
+    /* Cleanup effect: Hide current alert and prevent pending alert
+     * from showing after the component unmounts due to navigation */
+    useEffect(
+        () => () => {
+            hideAlert()
+            clearTimeout(alertTimeoutRef.current)
+        },
+        [hideAlert]
+    )
 
     return (
-        <>
-            <div
-                className={cx(classes.container, 'dashboard-scroll-container')}
-                data-test="outer-scroll-container"
-            >
-                <DashboardsBar
-                    expanded={controlbarExpanded}
-                    onExpandedChanged={onExpandedChanged}
+        <div
+            className={cx(classes.container, 'dashboard-scroll-container')}
+            data-test="outer-scroll-container"
+        >
+            <DashboardsBar />
+            <DashboardContainer covered={false}>
```

Comment:
Seems safe to remove the `covered` property. This is the only place it is passed and it's always false.

Reply:
So you're not going to remove `covered` from DashboardContainer in this PR? Seems most relevant to this PR since `covered` was all about the expanded control bar.